pom.xml

```diff
@@ -71,11 +71,11 @@
         <hamcrest.version>2.2</hamcrest.version>
         <logback.version>1.2.3</logback.version>
         <modernizr.version>2.8.3-1</modernizr.version>
-        <mockito.version>3.11.1</mockito.version>
+        <mockito.version>3.11.2</mockito.version>
         <popperjs.version>1.14.1</popperjs.version>
         <servlet-api.version>3.1.0</servlet-api.version>
         <slf4j.version>1.7.25</slf4j.version>
-        <wicket.version>9.3.0</wicket.version>
+        <wicket.version>9.4.0</wicket.version>
         <wicketstuff.version>9.3.0</wicketstuff.version>
```

Comment:
Shall we wait for WicketStuff 9.4.0 and then merge ? :-)

Reply:
Probably yes :)
BUT currently I can't start a release due to lots of Javadoc issues :(((

Reply:
Javadoc issues in WicketStuff ?
Why suddenly javadoc causes us so much troubles ?!
I still cannot figure out why `mvn javadoc:aggregate` does not work in Wicket 9.x

Reply:
The is the same issue (from different angle) :))

```
org.apache.maven.reporting.MavenReportException: 
Exit code: 1 - javadoc: error - Error fetching URL: https://ci.apache.org/projects/wicket/apidocs/9.x/

Command line was: /usr/lib/jvm/java-11-openjdk-amd64/bin/javadoc -J-Xmx256m -J-Xms128m @options @packages

Refer to the generated Javadoc files in '/home/solomax/work/wicketstuff/wicketstuff-datetime-yui-parent/wicketstuff-datetime-yui-examples/target/apidocs' dir.
```

I have created https://issues.apache.org/jira/browse/INFRA-22089
Let's see if it will help :)

Reply:
Oh, my!
I think you can overcome this issue by using failOnError=false - https://github.com/apache/wicket/blob/1f95c72f00dfb1a57d5f37d5b1e2b78bfbfd31dd/pom.xml#L1012

Reply:
This is already the case - https://github.com/wicketstuff/core/blob/29d92263dc5332cc850b2fd05bb5b4ac455bbde6/pom.xml#L1010
Well, try by removing the `<link>`

Reply:
Well
I can make the build green and release, but this will be sort of imperfect build
I would prefer to spend some time on Jenkins first :)